causely/manifest.json

```diff
@@ -0,0 +1,57 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "0ac3b5eb-46d2-4091-a001-e95dc5782609",
+  "app_id": "causely",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Causal AI platform automatically captures, represents and analyzes causality.",
+    "title": "Causely",
+    "media": [
+      {
+        "media_type": "image",
```

Comment:
Can you add an image of data in Datadog?

Reply:
added a screenshot of the dashboard with an event, thank you!